src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/DefendantPinLetterContentProviderTest.java

```diff
@@ -140,4 +141,11 @@ public void shouldProvideHmctsEmail() {
         assertThat(content).containsEntry("hmctsEmail", STAFF_NOTIFICATIONS_RECIPIENT);
     }
 
+    @Test
+    public void shouldProvideHmctsNewFeatureConsent() {
+        Map<String, Object> content = provider.createContent(claim, DEFENDANT_PIN);
+
+        assertThat(content).containsEntry(NEW_FEATURES, Boolean.TRUE);
```

Comment:
`assertTrue(content).containsEntry(NEW_FEATURES) ` ?

However Test will never fail once Feature is Permanent.  An additional Negative Test for absence of text, is maybe more helpful.

Reply:
I have added the negative test case. Thanks.